st2auth/st2auth/backends/__init__.py

```diff
@@ -59,4 +63,6 @@ def get_backend_instance(name):
     else:
         kwargs = {}
 
-    return cls(**kwargs)
+    cls = manager.driver
+    cls_instance = cls(**kwargs)
```

Comment:
Yeah, it's a fatal exception (e.g. you pass invalid option / keyword argument to the backend class) which means it should propagate all the way up.

The error is not the most user-friendly right now, but for more friendly error messages we would really need a (json) schema for each backend configuration which is out of scope now (it's more of a new feature).